ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-presubmits.yaml

```diff
@@ -15,7 +15,7 @@ presubmits:
     skip_branches:
     - gh-pages
     - release-1.20
-    skip_if_only_changed: ^docs/|\.(md|adoc)$|^(README|LICENSE)$
+    skip_if_only_changed: ^(?:docs|tutorials|logo|\.github)/|\.md$|^(?:\.gitignore|OWNERS|SECURITY_CONTACTS|PROJECT|LICENSE)$
```

Comment:
any reason to not group the two capture groups together?

Reply:
No. No particular reason. Do you anticipate any issues?